components/admin_console/custom_url_schemes_setting.jsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import * as Utils from 'utils/utils';
+
+import Setting from './setting';
+
+export default class CustomUrlSchemesSetting extends React.Component {
+    static get propTypes() {
+        return {
+            id: PropTypes.string.isRequired,
+            value: PropTypes.array.isRequired,
+            onChange: PropTypes.func.isRequired,
+            disabled: PropTypes.bool,
+            setByEnv: PropTypes.bool.isRequired,
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: this.arrayToString(props.value),
+        };
+    }
+
+    stringToArray = (str) => {
+        return str.split(',').map((s) => s.trim()).filter(Boolean);
+    };
+
+    arrayToString = (arr) => {
+        return arr.join(',');
+    };
+
+    handleChange = (e) => {
+        const valueAsArray = this.stringToArray(e.target.value);
+
+        this.props.onChange(this.props.id, valueAsArray);
+
+        this.setState({
+            value: e.target.value,
+        });
+    };
+
+    render() {
+        const label = Utils.localizeMessage('admin.customization.customUrlSchemes', 'Custom URL Schemes:');
+        const helpText = Utils.localizeMessage(
+            'admin.customization.customUrlSchemesDesc',
+            'Allow message text to link if it begins with any of the URL schemes listed here. By default, the following schemes will create links: "http", "https", "ftp", "tel", and "mailto".'
```

Comment:
Update to: Allow message text to link if it begins with any of the comma-separated URL schemes listed here.